.github/workflows/deploy-release.yml

```diff
@@ -28,3 +28,14 @@ jobs:
         :t-intensifies: `prod` updated to release `${{ github.event.release.tag_name }}`
       failure-message: |
         release `${{ github.event.release.tag_name }}` failed to update `prod`
+
+  sentry:
+    needs: [release]
+    uses: getsentry/action-release@v1.7.0
+    env:
+      SENTRY_AUTH_TOKEN: ${{ secrets.SENTRY_AUTH_TOKEN }}
+      SENTRY_ORG: mbtace
+      SENTRY_PROJECT: mbta-dotcom
+    with:
+      environment: production
```

Comment:
I believe we want `prod` rather than `production` here, to match the sentry config's `included_environments`.

Reply:
probably a typo from years ago 🥲